src/devices/AD5328/README.md

```diff
@@ -0,0 +1,14 @@
+# AD5328 - Digital to Analog Convertor
+AD5328 is an Digital-to-Analog converter (DAC) with 12 bits of resolution.
+
+## Usage
+```C#
+    var spisettings = new SpiConnectionSettings(0, 1)
    {
        Mode = SpiMode.Mode2
    };
```

Comment:
Formatting issues

Reply:
I think I fixed this not 100% sure though